content/docs/reference/install.md

```diff
@@ -12,9 +12,16 @@ NOTE: To update this page with a new binary release, do the following:
 - Update `content/reference/changelog.md`
 -->
 
-This page contains detailed instructions on installing Pulumi.
+This page contains detailed instructions for [downloading](#download) and [installing Pulumi](#install-pulumi) on your machine.
 
-For a streamlined walkthrough of getting started with Pulumi, including installing Pulumi and prerequisites, see the [Get Started]({{< relref "/docs/quickstart" >}}) guide.
+## Download
+
+The current stable version of Pulumi is **{{< latest-version >}}**. For a full history of prior versions, including
+release notes and direct download links, visit the <a href="{{< relref "/docs/reference/changelog" >}}">Changelog</a> page.
+Pulumi is packaged as a tarball for Linux and macOS, and as a zip archive for Windows.
```

Comment:
The downside of moving this Download section to the top of the page is that it implies that this is what users should be doing. But most users should not be downloading these and installing manually (download/manual-install is a more advanced scenario). For macOS, the majority of users should be using Homebrew, for Linux the installation script, and for Windows the installation script (though, the Windows installation script experience isn't great and may trigger anti-virus -- I believe Paul was looking into Chocolately, which might become our preferred approach there when it's available).

So I worry a little that by having a Download section at the top, folks are going to first try to download a version for their platform, without having read the rest of this page, and then be left scratching their heads with how to install the binary manually. When instead, they should have used one of the preferred methods for their platform.

Reply:
Instead of this section, what do you think of just adding a sentence to the Manual Installation for each of the platforms?

For macOS, something like:

```diff
- 1. Download [Pulumi {{< latest-version >}} for macOS](https://get.pulumi.com/releases/sdk/pulumi-v{{< latest-version >}}-darwin-x64.tar.gz).
+ 1. Download [Pulumi {{< latest-version >}} for macOS](https://get.pulumi.com/releases/sdk/pulumi-v{{< latest-version >}}-darwin-x64.tar.gz). Prior versions and release notes are available [here]({{< relref "/docs/reference/changelog" >}}).
```

And do something similar for Linux and Windows.

Reply:
Ah yes, we can do something like that for now. But as we release more major versions, we might want to consider adding a "Release Notes" link to the left nav, or a "View Release Notes" button at the top of the Download and Install page.

Are we ok with leaving this scenario vague though?
![Screenshot from 2019-07-31 09-05-25](https://user-images.githubusercontent.com/5844484/62228397-6e51a980-b372-11e9-9201-e42163b85ff1.png)

Does that get triggered often?

Reply:
I have one more question, @justinvp : Did we mean to add `either` to the following instruction?
> 2. Unzip the tarball and either copy the binaries in the `pulumi` directory on your `$PATH`.

We have this sentence under "Manual Installation" for macOs and Linux.

Reply:
> Are we ok with leaving this scenario vague though?

Which scenario?

> Does that get triggered often?

Does what get trigged often? The "(choose one)" option? I doubt it's ever triggered, unless someone selects it manually. Next milestone I am going to change the install step of the Get Started guide to show tabs for Linux, macOS, and Windows (#1394). What do you think of switching this to be tabs as well (instead of the drop down)?

Reply:
Yes. Will the tabs have the installation options then verify and uninstall will be outside of the tabbed content? I think tabs might look cleaner.

Reply:
> Did we mean to add `either` to the following instruction?

Probably not. It should probably be something like:

> 2. Unzip the tarball and copy the binaries to a `pulumi` directory on your `$PATH`.

But feel free to rewrite it so it actually makes sense 😉

> Will the tabs have the installation options then verify and uninstall will be outside of the tabbed content?

I think so.